cloud-function/src/handlers/proxy-bsa.ts

```diff
@@ -40,6 +47,8 @@ export async function proxyBSA(req: Request, res: Response) {
       userAgent
     );
 
+    payload.plusAvailable = plusAvailable;
```

Comment:
Nit: Do we really need the `plusAvailable` variable, or can we not directly assign here?